docs/root/operations/admin.rst

```diff
@@ -84,6 +84,7 @@ modify different aspects of the server:
       cx_total, Counter, Total connections
       cx_active, Gauge, Total active connections
       cx_connect_fail, Counter, Total connection failures
+      rq_hedge_abandoned, Counter, Total hedged requests that were canceled and abandoned due to accepting another response.
```

Comment:
Should we also have a counter on hedged request total? WDYT?

Reply:
will add one